src/commands/core/help.ts

```diff
@@ -39,7 +39,7 @@ export default class extends Command {
             const response = message.translate('core/help:INVALID', {
                 name: parameters
             });
-            if (!message.embeddable) return message.error(response);
+            if (!message.embeddable) return message.error(response, undefined, { allowedMentions: { users: [message.author.id]}});
```

Comment:
I'm not seeing how `TypicalMessage#error` currently takes in options? Only parameters it seems to take is `content` and `embed`.